lib/google_sheets_service.rb

```diff
@@ -0,0 +1,19 @@
+require "dotenv/load"
+require "google/apis/sheets_v4"
+require "googleauth"
+
+class GoogleSheetsService
+  def initialize
+    @service = Google::Apis::SheetsV4::SheetsService.new
+    @service.client_options.application_name = "dxw Glossary"
+    @service.authorization = Google::Auth::ServiceAccountCredentials.make_creds(
+      json_key_io: StringIO.new(ENV.fetch("GOOGLE_CLIENT_SECRET")),
+      scope: Google::Apis::SheetsV4::AUTH_SPREADSHEETS_READONLY
+    )
+  end
+
+  def spreadsheet_values(spreadsheet_id: ENV.fetch("SPREADSHEET_ID"))
+    range = "Glossary!A2:G"
```

Comment:
Optional refactoring: I like the fact the `SPREADSHEET_ID` is configurable. We _could_ also make the range here configurable so the user could set the sheet and cell range in the same way. If we want to make it easy for others to use then this would make it easier for them to use their own sheets.